packages/flutter/test/native/android_envelope_sender_test_real.dart

```diff
@@ -40,7 +40,7 @@ void main() {
       expect(() => sender.close(), returnsNormally);
     });
 
-    test('warns and drops after close', () async {
+    test('buffers envelopes after close', () async {
```

Comment:
Is buffering correct after we close the sender, or should we just handle the deferred init case?